src/nodes/chunk_dispatch_state.c

```diff
@@ -127,43 +127,44 @@ chunk_dispatch_exec(CustomScanState *node)
 	if (cis->hyper_to_chunk_map != NULL)
 		slot = execute_attr_map_slot(cis->hyper_to_chunk_map->attrMap, slot, cis->slot);
 
-	if (cis->compress_state != NULL)
+	if (cis->compress_info != NULL)
 	{
 		/*
 		 * When the chunk is compressed, we redirect the insert to the internal compressed
 		 * chunk. However, any BEFORE ROW triggers defined on the chunk have to be executed
 		 * before we redirect the insert.
 		 */
-		if (cis->orig_result_relation_info->ri_TrigDesc &&
-			cis->orig_result_relation_info->ri_TrigDesc->trig_insert_before_row)
+		if (cis->compress_info->orig_result_relation_info->ri_TrigDesc &&
+			cis->compress_info->orig_result_relation_info->ri_TrigDesc->trig_insert_before_row)
 		{
 			bool skip_tuple;
-			skip_tuple = !ExecBRInsertTriggers(estate, cis->orig_result_relation_info, slot);
+			skip_tuple =
+				!ExecBRInsertTriggers(estate, cis->compress_info->orig_result_relation_info, slot);
 
 			if (skip_tuple)
 				return NULL;
 		}
 
 		if (cis->rel->rd_att->constr && cis->rel->rd_att->constr->has_generated_stored)
-			ExecComputeStoredGeneratedCompat(cis->orig_result_relation_info,
+			ExecComputeStoredGeneratedCompat(cis->compress_info->orig_result_relation_info,
```

Comment:
Codecov says that this line is never triggered by tests. Would it be difficult to fix it?

Reply:
This PR only makes a syntactic change. I'll look into adding a test case in another PR.